portfolio/src/main/webapp/index.html

```diff
@@ -5,63 +5,65 @@
-      <div id="intro-button-one" style="display:inline-block">
+        <div id="intro-button-one" style="display:inline-block">
         <div class="mdc-touch-target-wrapper" style="display:block;">
-          <button class="mdc-button mdc-button--outlined" onclick="intro()" style="color:white;">  
-	     <span class="mdc-button__ripple">Intro</span> 
-	  </button>
+            <button class="mdc-button mdc-button--outlined" onclick="intro()" style="color:white;">  
+                <span class="mdc-button__ripple">Intro</span>
+            </button>
         </div>
         <div class="mdc-touch-target-wrapper" style="display:block;">
-           <button class="mdc-button mdc-button--outlined" onclick="tech()" style="color:white;">  
-	      <span class="mdc-button__ripple">Tech Skills</span> 
-           </button>
+            <button class="mdc-button mdc-button--outlined" onclick="tech()" style="color:white;">  
+                <span class="mdc-button__ripple">Tech Skills</span>
+            </button>
         </div>
         <div class="mdc-touch-target-wrapper" style="display:block;">
-	   <button class="mdc-button mdc-button--outlined" onclick="interests()" style="color:white;">  
-	       <span class="mdc-button__ripple">Interests</span>
-	   </button>
+            <button class="mdc-button mdc-button--outlined" onclick="interests()" style="color:white;">  
+                <span class="mdc-button__ripple">Interests</span>
+            </button>
         </div>
-      </div>
-      <div id="basic-profile" style="display:inline-block;">
-        <img src="/images/PROFILE_PIC.JPG">
-      	<h1>Michaela Murray's Portfolio</h1>
-      	<div id="intro">
-      	  <ul style="list-style-type:none;">
-            <li>Stanford Class of 2022</li> 
-            <li>Major: Computer Science</li>
-            <li>STEP Intern @ Google</li>
-	  </ul>
         </div>
-      </div>
-      <div id="intro-button-two" style="display:inline-block;">
-	<div class="mdc-touch-target-wrapper" style="display:block;">
-	   <button class="mdc-button mdc-button--outlined" style="color:white;">  
-	      <span class="mdc-button__ripple">Blog</span>
-	   </button>
+        <div id="profile-wrapper" style="display:inline-block;">
+        <div id="basic-profile">
+            <div id="wrapper">
+                <img id="profile" src="/images/PROFILE_PIC.JPG">
+            </div>
+            <h1>Michaela Murray's Portfolio</h1>
+            <div id="intro">
+                <ul style="list-style-type: none;">
+                <li>Stanford Class of 2022</li> 
+                <li>Major: Computer Science</li>
+                <li>STEP Intern @ Google</li>
+                </ul>
+            </div>
+        </div>
+        </div>
+        <div id="intro-button-two" style="display: inline-block;">
+        <div class="mdc-touch-target-wrapper" style="display: block;">
+            <button class="mdc-button mdc-button--outlined" onclick="blog()" style="color:white;">  
```

Comment:
nit - revert this change related to the blog.

Reply:
I took out the blog function! (I believe the button was there before) Thanks!